src/views/Hearing/index.js

```diff
@@ -225,12 +226,14 @@ function groupSections(sections) {
     const sectionGroup = find(sectionGroups, group => section.type === group.type);
     if (sectionGroup) {
       sectionGroup.sections.push(section);
+      sectionGroup.n_comments += section.n_comments;
```

Comment:
As said, the child component does not mutate a prop. This is constructing a new prop for a child component, and it is done within a single function in a single component.